src/main/java/com/github/jcustenborder/kafka/connect/snmp/PrivacyProtocol.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright © 2021 Elisa Oyj
+ * Copyright © 2017 Jeremy Custenborder (jcustenborder@gmail.com)
```

Comment:
Also this copyright is not inherited to new work in new files. You can remove it from this file.